### DISCUSSION TITLE
Online meeting URL's can't be deleted

### DISCUSSION THREAD

**Discussion by ljread** [code4recovery/12-step-meeting-list#1353] [Q&A] [answered]
We just moved to a new web host provider. Don't know if this issue is related.

When trying to remove a URL for an online or hybrid meeting, the URL repopulates after pressing the update button (which sometimes requires pressing twice to get an update notification.) Changing or adding a URL works. All other editing functions appear to work. Deleting the meeting and adding back works. TSML appears to display and function normally.

I had thought to delete and reinstall the TSML plugin but would rather avoid this if I can. Any ideas to fix without going to this extreme would be appreciated.

I will try to logon to the old server and see if the issue occurs there as well. I also see there is a new version of TSML, so I will update and hope that fixes my issue.

Wordpress version: 6.4.3
TSML version: 3.14.30

Thanks, John Read

**Comment by joshreisner** [comment]
thanks for the report, definitely seems to be an issue, we're on it!

**Comment by ljread** [comment]
Not been able to log on to old server yet. Update to 3.14.31 did not help. Thanks for the quick response, Josh

**Reply by kiyote33** [reply]
Hi John @ljread , we've found that the bug crept into our code in the 3.14.29 release. While we work on resolving this issue, you could choose to rollback your code to the 3.14.28 release (I use wp_rollback to do that) and you should be able to delete URLs again. Sorry for the inconvenience this issue is causing you.

**Comment by ljread** [comment]
Thanks Wayne for the prompt reply. Right now it's a minor problem. I'll probably wait for the fix, unless we have a run on groups dropping online meetings. If it becomes an issue I will rollback as you suggested.

Thanks for all the good work you all do.

**Comment by joshreisner** [comment, marked as answer]
ok we've released `3.14.32` which should address this issue, let us know if you still have problems!

we will follow up on having to click Publish twice in #1357

**Comment by ljread** [comment]
I will update later this morning and post my results. Thanks again for the prompt reply. You guys are great!

**Comment by ljread** [comment]
Updating to 3.14.32 fixed the issue with URL's repopulating after deletion. Thanks again! Will close this issue since Publish twice is covered in #1357
